example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -539,7 +539,7 @@
 		};
 		97C147071CF9000F007C117D /* Release */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = 2FBF59F345E05927E16F325B /* Pods-Runner.release.xcconfig */;
+			baseConfigurationReference = 7AFA3C8E1D35360C0083082E /* Release.xcconfig */;
```

Comment:
Any idea why this changed? It's just the example but just curious

Reply:
I should have added a comment, my bad. 
This is a revert of a misguided update from the previous PR: https://github.com/RevenueCat/purchases-flutter/pull/127/files

Turns out that when you manually `pod install`, you get a warning from cocoapods, so I made the change on the previous pr to fix it... 
But the warning was wrong. 
Essentially the warning is cocoapods saying "you should be using the xcconfig from cocoapods", but the ones generated by flutter already include the ones in cocoapods.